blackbox/docs/general/builtins/aggregation.rst

```diff
@@ -69,9 +69,7 @@ The synopsis of an aggregate expression is one of the following
    aggregate_function ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE condition ) ]
 
 where ``aggregate_function`` is a name of an
-:ref:`aggregate function <aggregate-functions>`.
-
-and ``expression`` is a column reference, scalar function or literal.
+:ref:`aggregate function <aggregate-functions>` and ``expression`` is a column reference, scalar function or literal.
```

Comment:
did it intentionally to have some kind of visual separation between expression args, but probably merging in this way is ok too. Could you please wrap the line to have the length that is not longer than of 80 chars?

Reply:
oh, sorry. I was confused because there was a full stop in between.